Model/Sales/Package.php

```diff
@@ -102,19 +102,35 @@ public function addWeight(float $weight)
     }
 
     /**
-     * @return int
+     * @return float
+     */
+    public function getMaxMailboxWeight(): float
+    {
+        return $this->maxMailBoxWeight;
+    }
+
+    /**
+     * @param float $max_weight
+     */
+    public function setMaxMailboxWeight(float $max_weight)
```

Comment:
```suggestion
    public function setMaxMailboxWeight(float $max_weight): void
```
Kan dit in jouw php versie?

Reply:
Zie ook andere functies

Reply:
done